README.md

```diff
@@ -1 +1,77 @@
-# Restaurant-Page
\ No newline at end of file
+# Project: Restaurant Page
+A simple restaurant application built with HTML, CSS & Javascript. 
+
+<br />
+<p align="center">
+
+  <h1 align="center">The Safa's</h1>
+
+  <p align="center">
+    This project is part of the Microverse curriculum and is done as an exercise in the Javascript section.
+    <br />
+    <br />
+    <a href="https://safaerden.github.io/Restaurant-Page/">View Demo</a>
+    ·
+    <a href="https://github.com/SafaErden/Restaurant-Page/issues">Report Bug</a>
+  </p>
+</p>
+
+![screenshot](./dist/assets/image/safas.JPG)
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Built With](#built-with)
+* [Contact](#contact)
+* [Acknowledgements](#acknowledgements)
+* [License](#license)
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+This project's purpose is to practive Javascript concepts like DOM manipulation and OOP with webpack.
+
+<!-- [![Product Name Screen Shot][product-screenshot]][screenshot-url] -->
+
+<!-- ABOUT THE PROJECT -->
+## Installation
+
+To use this app locally, this is what you need to:
+
+* [Download](https://github.com/SafaErden/Restaurant-Page/archive/development.zip) or clone this repo:
+  - Clone with SSH:
+  ```
+    git@github.com:SafaErden/Restaurant-Page.git
+  ```
+  - Clone with HTTPS
+  ```
+    https://github.com/SafaErden/Restaurant-Page.git
+  ```
+ 
```

Comment:
### [required change]
- After adding scripts-commands inside `package.json`, you should come to this file and add `webpack` commands on how to transpile the Javascript files (after cloning)
```
npm run build
```
(and you can precise that project be opened from `dist/` folder)
If you added the `start` option, you should additionally add commands to START the project and run it locally
```
 npm run start
```

(Cloning only as you wrote, isn't enough)